src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -388,6 +390,14 @@ export class ResultsViewPageStore {
                     }, false);
 
                 return anySelected;
+            },
+
+            get driverFilter() {
```

Comment:
I just realized that driverFilter is too general a name. It should be customBinaryDriverFilter, and for tiers lets change it to customTiersDriverFilter.